test/e2e/app-dir/logging/fetch-logging.test.ts

```diff
@@ -247,6 +234,22 @@ describe('app-dir - logging', () => {
           )
         })
 
+        it('should log requests for client-side navigations', async () => {
```

Comment:
Just reordered the test - to rule out that the flakiness could possibly be due to timeout on the last test